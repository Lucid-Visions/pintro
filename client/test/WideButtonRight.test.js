import React from "react";
import WideButtonRight from "../src/components/WideButtonRight";
import renderer from "react-test-renderer";

it('WideButtonRight renders correctly', () => {
    const tree = renderer
      .create(
      <WideButtonRight>
        value={"TEST VALUE"}
        source={require("../src/assets/arrow-right-white.png")}
      </WideButtonRight>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});

it('WideButtonRight renders correctly', () => {
    const tree = renderer
      .create(
      <WideButtonRight>
        value={"TEST VALUE"}
      </WideButtonRight>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});

it('WideButtonRight renders correctly', () => {
    const tree = renderer
      .create(
      <WideButtonRight>
        value={"TEST VALUE"}
        containerStyle={{
            fontFamily: "poppins-medium", 
            width: 350,
            marginTop: 30,
            backgroundColor: "#1A1A1A",
            flexDirection: 'row', 
            justifyContent: 'space-evenly'
        }}
        textStyle={{ 
            fontSize: 14, 
            fontFamily: "poppins-light", 
            color: 'lightgrey' 
        }}
      </WideButtonRight>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});

it('WideButtonRight renders correctly', () => {
    const tree = renderer
      .create(
      <WideButtonRight>
      </WideButtonRight>)
      .toJSON();
    expect(tree).toMatchSnapshot();
});
