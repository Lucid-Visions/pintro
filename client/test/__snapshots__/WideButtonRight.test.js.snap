// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WideButtonRight renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 25,
      "display": "flex",
      "height": 54.12,
      "justifyContent": "center",
      "marginLeft": "auto",
      "marginRight": "auto",
      "width": 400.88,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 10,
          "marginRight": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "black",
            "fontFamily": "poppins-medium",
            "fontSize": 15,
          }
        }
      />
    </View>
  </View>
</View>
`;

exports[`WideButtonRight renders correctly 2`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 25,
      "display": "flex",
      "height": 54.12,
      "justifyContent": "center",
      "marginLeft": "auto",
      "marginRight": "auto",
      "width": 400.88,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 10,
          "marginRight": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "black",
            "fontFamily": "poppins-medium",
            "fontSize": 15,
          }
        }
      />
    </View>
  </View>
</View>
`;

exports[`WideButtonRight renders correctly 3`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 25,
      "display": "flex",
      "height": 54.12,
      "justifyContent": "center",
      "marginLeft": "auto",
      "marginRight": "auto",
      "width": 400.88,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 10,
          "marginRight": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "black",
            "fontFamily": "poppins-medium",
            "fontSize": 15,
          }
        }
      />
    </View>
  </View>
</View>
`;

exports[`WideButtonRight renders correctly 4`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 25,
      "display": "flex",
      "height": 54.12,
      "justifyContent": "center",
      "marginLeft": "auto",
      "marginRight": "auto",
      "width": 400.88,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "marginLeft": 10,
          "marginRight": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "black",
            "fontFamily": "poppins-medium",
            "fontSize": 15,
          }
        }
      />
    </View>
  </View>
</View>
`;
